v03_pipeline/migrations/lookup/0003_add_sample_type.py

```diff
@@ -0,0 +1,93 @@
+import hail as hl
+import hailtop.fs as hfs
+
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.migration.base_migration import BaseMigration
+from v03_pipeline.lib.model import DatasetType, ReferenceGenome, SampleType
+from v03_pipeline.lib.paths import project_table_path
+
+logger = get_logger(__name__)
+
+
+class AddLookupSampleType(BaseMigration):
+    reference_genome_dataset_types: frozenset[
+        tuple[ReferenceGenome, DatasetType]
+    ] = frozenset(
+        (
+            (ReferenceGenome.GRCh37, DatasetType.SNV_INDEL),
+            (ReferenceGenome.GRCh38, DatasetType.SNV_INDEL),
+            (ReferenceGenome.GRCh38, DatasetType.MITO),
+        ),
+    )
+
+    @staticmethod
+    def migrate(
+        ht: hl.Table,
+        reference_genome: ReferenceGenome,
+        dataset_type: DatasetType,
+    ) -> hl.Table:
+        """
+        Adds sample_type to lookup ht global fields project_guids and project_families.
+        Assumes that only one project_ht exists for each project across both sample types.
+
+        Old Global fields:
+            'project_guids': array<str>
+            'project_families': dict<str, array<str>>
+            'updates': set<struct {
+                callset: str,
+                project_guid: str,
+                remap_pedigree_hash: int32
+            }>
+        New Global fields:
+            'project_guids': array<tuple (
```

Comment:
It might be worth renaming this field to something like `project_sample_types` to make it clearer what it is